lib/ansible/plugins/action/validate.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/python
```

Comment:
Tests are failing due to the presence of the shebang. Since this is a plugin the shebang should be omitted.